### PR TITLE
Make column schema for fields

### PR DESCRIPTION
**Pull request by ricardolsmendes**
**- What I did**
Added a feature that allows the Sisense Connector to synchronize Widget fields as schema columns in Google Data Catalog.

**- How I did it**
1. Refactored the `__make_schema_for_widget()` method of `scrape.DataCatalogEntryFactory` so it delegates fields and filters columns generation.
2. Created the `DataCatalogEntryFactory.__make_fields_column_for_widget()` method.
3. Created the `DataCatalogEntryFactory.__make_filters_column_for_widget()` method.
4. Added the `prepare.SisenseConnectorStringsHelper` temporary class (instructions on how to remove it in the future are available in the class docs).
5. Add unit tests to fully cover the above changes.

**- How to verify it**
Run the unit tests.

**- Description for the changelog**
Added a feature that allows the Sisense Connector to synchronize Widget fields as schema columns in Google Data Catalog.

PS: This PR is part of the effort to deliver #70.

### CODE REVIEW

File: google-datacatalog-sisense-connector/src/google/datacatalog_connectors/sisense/prepare/sisense_connector_strings_helper.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+from google.datacatalog_connectors.commons import prepare
+
+
+class SisenseConnectorStringsHelper:
+    """
+    Temporary class providing the logic to format column names currently
+    required by the Entry and Tag factories.
+
+    TODO
```
Comment (mesmacosta) [inline google-datacatalog-sisense-connector/src/google/datacatalog_connectors/sisense/prepare/sisense_connector_strings_helper.py:29]: Turn this TODO into a cleanup issue in the commons repo?

**Review by mesmacosta — APPROVED** [review]
LGTM! Sorry for the delay.

File: google-datacatalog-sisense-connector/src/google/datacatalog_connectors/sisense/prepare/sisense_connector_strings_helper.py (hunk 1 above)
Comment (ricardolsmendes) [reply google-datacatalog-sisense-connector/src/google/datacatalog_connectors/sisense/prepare/sisense_connector_strings_helper.py:29]: Sure! Would you like to take this task? 😃

Comment (mesmacosta) [reply google-datacatalog-sisense-connector/src/google/datacatalog_connectors/sisense/prepare/sisense_connector_strings_helper.py:29]: can't atm :) my suggestion was more to track that.

Comment (ricardolsmendes) [reply google-datacatalog-sisense-connector/src/google/datacatalog_connectors/sisense/prepare/sisense_connector_strings_helper.py:29]: No worries. I've just filed a feature request: https://github.com/GoogleCloudPlatform/datacatalog-connectors/issues/54.

**Pull request merged**